src/components/recordset/recordset.tsx

```diff
@@ -367,7 +369,8 @@ const RecordsetInner = ({
     }
 
     // handle the scrollable container
-    const resizeSensors = attachContainerHeightSensors(parentContainer, parentStickyArea);
+    const resizeSensors = attachContainerHeightSensors(parentContainer, parentStickyArea, undefined, setTopPanelDimensions);
```

Comment:
The more I think about it, the more I don't like this solution. Since the callback is called on each height or width change, React is rerendering recordset component and all its children on each height/width change. So this solution would never work on record page, and it can slow the page down on long recordset pages.

As I mentioned before, we're just manipulating the styles attached to the elements, so there's no reason to involve React state in this. Instead of calling `setTopPanelDimensions` and passing the state variable to `RecordsetTable`, is there anyway that we can pass the `top` value from here? Maybe attach it as a `data-*` attribute to an element and then grab it from that page?